preamble.tex

```diff
@@ -70,4 +60,16 @@
 \newcommand{\TODO}{\hl{\textbf{TODO}}\xspace}
 
 % Pathing
-\newcommand{\subDir}[1]{chapters/chapter\currChapter/chapter\currChapter-#1}
\ No newline at end of file
+\newcommand{\subDir}[1]{chapters/chapter\currChapter/chapter\currChapter-#1}
+
+% Margins & layout
+\setlrmarginsandblock{1in}{*}{1}
+\setulmarginsandblock{1in}{*}{1}
+\renewcommand*{\cftdotsep}{1} % adjustment for the ToC
+\setpnumwidth{2em}
+\setrmarg{4em}
+\checkandfixthelayout[nearest]
+
+% Debugging v/hbox in logs
+\showboxdepth=\maxdimen
+\showboxbreadth=\maxdimen
```

Comment:
Do we need to include debugging on master?

Reply:
welp, I forgot to comment these!